### ISSUE TITLE
Can we skip user creation in the Mac App?

### ISSUE THREAD

**Issue by camsaul** [metabase/metabase#1185]
I'm sure this has already been discussed but I'm out of the loop WRT this so bear with me

It would be hella rad if I could just fire up the Mac App for the first time and jump straight to the add DB screen. If I use `pgadmin2` or `MySQLThing` or `emacs-db-mode` on my computer I don't need an account to use it. I'm not sure the account serves much of a purpose.

This could be something configurable via a special env var that the Mac app runs the uberjar with or something like that

⬇️  **Please click the 👍 reaction instead of leaving a `+1` or 👍  comment**

**Comment by pdurbin** [comment]
I upgraded Metabase on my Mac to 0.13.2.0 the other day (as prompted) and I got logged out. I couldn't remember the password I used. I tried the password reset button but despite #1565 saying it works now I couldn't get it to work. I ended up blowing away `~/Library/Application Support/Metabase` and `Library/Caches/com.metabase.Metabase` and now I have the opportunity to create an account and start kicking the tires on Metabase some more. This is rather a lot of trouble and I would prefer that I don't have to create an account at all for the Mac app. [pgAdmin](http://pgadmin.org) (and Chrome and basically every Mac app) doesn't ask me to create an account and it would be fantastic if Metabase didn't either.

Let me back up and say that I think it's **wonderful** that you offer a desktop app in the first place but some sort of "this is just me on my laptop" mode would be great where an account doesn't have to be created at all. No one is going to use Metabase on my laptop but me.

Despite this, Metabase seems great! In another issue I'm trying to build it locally: #1690 :smile:

**Comment by styts** [comment]
As mentioned above, it was sufficient for me to `rm -i ~/Library/Application\ Support/Metabase/metabase.*.db` to reset the password in the local app.

**Comment by camsaul** [comment]
@styts You can just use the password reset tool that comes with the Mac App, you can find it in the menu. No need to delete your database.

At any rate I think password reset is a bit off-topic for this issue. Feel free to open a new one if you're running into problems with it.

**Comment by jakob** [comment]
I just wanted to mention that I support this idea. I evaluated Metabase in response to [this pull request](https://github.com/PostgresApp/PostgresApp/pull/416), and the signup flow was a real turn-off.

When I started with PostgreSQL and evaluated software to go with it, I threw out a lot of apps because setting them up was such a hassle.

It's also unclear whether any of the data from the signup flow is transmitted to a web service or just stored locally. In my experience as a desktop app developer, having an app that runs locally and does not transmit data to a web service is an important differentiator.
